src/components/TileGrid/TileEdit.tsx

```diff
@@ -70,7 +75,7 @@ function TileEditMenu({ visible, setVisible, options, icons }: ITileEditMenu) {
               key={option.name}
```

Comment:
The key should be pushed up one level. Instead of `<> ... </>`, use `<React.Fragment key={...}>...</React.Fragment>`